Sources/CryptorRSA/CryptorRSAKey.swift

```diff
@@ -683,7 +683,7 @@ public extension CryptorRSA {
                 range: all
             )
 			
-			let keys = matches.flatMap { result -> PublicKey? in
+			let keys = matches.compactMap { result -> PublicKey? in
```

Comment:
I think this rename happened in Swift 4.1 so this will break on Swift 4.0.3.  We'll need to `#if swift` it.